rosbag2_compression/src/rosbag2_compression/zstd_decompressor.cpp

```diff
@@ -31,6 +31,8 @@ namespace
 
 // String constant used to identify ZstdDecompressor.
 constexpr const char kDecompressionIdentifier[] = "zstd";
+constexpr const uint32_t kMagicNumberLittleEndian = 0xFD2FB528;
+constexpr const uint32_t kMagicNumberBigEndian = 0x28B52FFD;
```

Comment:
Could we get this from the lib itself? Better not to duplicate the information.

Reply:
yes, via `ZSTD_MAGICNUMBER`